### ISSUE TITLE
Tela Principal

### ISSUE THREAD

**Issue by lucasmontano** [lucasmontano/first-how-to-build-an-app#5]
Como usuário deste aplicativo, que já cadastrou um medo, eu devo ser apresentado a uma tela que me ajude a superar meus medos e alcançar meus objetivos. 

## Fluxo de navegação
Esta tela deve ser a tela inicial do aplicativo, quando o usuário já cadastrou um medo. Se o usuário não cadastrou um medo ainda, esta tela é exibida após concluir o fluxo de cadastro do primeiro medo.

## Definição da tela
No topo da tela devemos exibir um item de conteúdo que ajude o usuário a superar seus medos. Nesta primeira versão, o usuário vai receber um conteúdo a cada dois dias 

### Definição do conteúdo
- um disclaimer deve aparecer abaixo do conteúdo explicando que o usuário vai receber um novo conteúdo a cada dois dias
- o conteúdo deve possuir uma imagem e um título
- ao clicar na imagem o usuário deve ter acesso ao conteúdo
--- Reproduzir um video se for um video
--- Abrir um link se for um link
--- Abrir um app / mandar para Google Play se for um app

### Lista dos medos

Para contextualizar o usuário os medos já cadastrados devem estar listados abaixo do conteúdo. Ao clicar no medo o usuário deve ser redirecionado para a Tela de Detalhes do Medo. 

### Elemento principal da tela
- Conteúdo

### Ação principal da tela ([FAB](https://material.io/design/components/buttons-floating-action-button.html))
- Cadastro de novo medo

## Wireframe
![Untitled design-3](https://user-images.githubusercontent.com/7559318/55292495-e1aad500-53eb-11e9-8f1d-9cfcea2f6632.png)

**Comment by gabe351** [comment]
Olá Lucas, você poderia criar um módulo de base para termos uma noção de como o projeto será estruturado (em arquitetura)

**Comment by lucasmontano** [comment]
@gabe351 claro! Vou criar a estrutura inicial do projeto essa semana, e vai estar no video de segunda. Vocês já vão conseguir acompanhar antes de segunda por aqui. Antecipando o assunto: vou abordar MVVM (usando ViewModel, [LiveData](https://developer.android.com/topic/libraries/architecture/livedata) e UiStates)
